providers/google-compute-engine/src/main/java/org/jclouds/googlecomputeengine/compute/options/GoogleComputeEngineTemplateOptions.java

```diff
@@ -280,4 +291,87 @@ public GoogleComputeEngineTemplateOptions runScript(String script) {
    public GoogleComputeEngineTemplateOptions blockOnComplete(boolean blockOnComplete) {
       return GoogleComputeEngineTemplateOptions.class.cast(super.blockOnComplete(blockOnComplete));
    }
+
+   public boolean canIpForward() {
+      return canIpForward;
+   }
+
+   /**
+    * @ee https://cloud.google.com/compute/docs/networking#canipforward
+    * @return
+    */
+   public void canIpForward(boolean enable) {
+      this.canIpForward = enable;
+   }
+
+   /**
+    * @see #getDisks()
+    * @see org.jclouds.googlecomputeengine.domain.templates.InstanceTemplate.AttachDisk
+    */
+   public GoogleComputeEngineTemplateOptions addDisk(AttachDisk disk) {
+      this.disks.add(disk);
+      return this;
+   }
+
+   public void autoCreateDisk(final AutoCreateDiskOptions diskOptions) {
+      this.autoCreateDisks.add(diskOptions);
+   }
+
+   public void autoCreateDisks(final List<AutoCreateDiskOptions> autoCreateDisks) {
+      this.autoCreateDisks.addAll(autoCreateDisks);
```

Comment:
Also, rename it to `addAutoCreateDisks` or similar, to avoid the confusion that this method would just assign the passed disks.